fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/type/NewNumberParmBehaviorUtil.java

```diff
@@ -236,15 +251,41 @@ public static void buildEqualsRangeClause(WhereFragment whereClauseSegment, Stri
     public static void buildApproxRangeClause(WhereFragment whereClauseSegment, String tableAlias,
             String columnBase, BigDecimal lowerBound, BigDecimal upperBound, BigDecimal value) {
         BigDecimal factor = value.multiply(FACTOR);
+        BigDecimal approximateLowerBound = lowerBound.subtract(factor);
+        BigDecimal approximateUpperBound = upperBound.add(factor);
 
         whereClauseSegment.leftParen();
-        whereClauseSegment.col(tableAlias, columnBase).gte().bind(lowerBound.subtract(factor)); // -10% of the Lower Bound
+        
+        whereClauseSegment.col(tableAlias, columnBase + _HIGH).gte().bind(approximateLowerBound);
         whereClauseSegment.and();
-        whereClauseSegment.col(tableAlias, columnBase).lt().bind(upperBound.add(factor)); // +10% of the Upper Bound
+        whereClauseSegment.col(tableAlias, columnBase + _LOW).lte().bind(approximateUpperBound);
+        
         whereClauseSegment.or();
-        whereClauseSegment.col(tableAlias, columnBase + _LOW).lte().bind(upperBound);
+        whereClauseSegment.col(tableAlias, columnBase).isNull();
```

Comment:
I've stared at this SQL for a while, it feels decently complex, could you inline some descriptions of the conditions being checked.  E.g. an example inline.  

It think it's OK, it'd just be great to not recreate the knowledge next time we're in the code.

Reply:
added comments with descriptions and example